Cleate messages controller

shiratori-sh opened the pull request:
#WHAT
Chatspaceチャット画面のビューファイル作成
/Users/shogo_shiratori/Documents/スクリーンショット 2019-03-11 17.15.28.png
#WHY
Hamlやscssの技術向上のため。
視覚的に操作するアプリケーションのため。

shiratori-sh committed:
Cleate-messages-controller

Create messages_controller
Edit messages_controller indexアクション

shiratori-sh committed:
create-messages-controller

index.html.haml作成
application.css→application.scss
application.scss 編集
リセットcssファイル作成「_reset.scss」

shiratori-sh committed:
create-messages-controller

_variable.scssファイル作成
messages_controllerにIndexアクションのルーティング
gem 'font-awesome-rails’追加

shiratori-sh committed:
Cleate-messages-controller

_variable.scssに変数定義
application.scssで反映するファイル指定。
message.scssとmessage/index.html.hamlファイル編集
views/layoutsのファイルの不要コメントアウト解除

shiratori-sh commented:
<img width="1677" alt="スクリーンショット 2019-03-11 17 15 28" src="https://user-images.githubusercontent.com/48270550/54109433-01209480-4422-11e9-9617-2c8908a61979.png">

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -1,11 +1,12 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットは修正しましょう。他の箇所も同様です

shiratori-sh replied:
ハッシュロケットの訂正しました。
作業ファイル：application.html.haml、index.html.haml

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -1,8 +1,8 @@
-\!!!
```
mailerファイルは使用しないので削除しましょう

shiratori-sh replied:
ファイルを削除しました。

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -1 +1 @@
-\= yield
+= yield
```
mailerファイルは使用しないので削除しましょう

shiratori-sh replied:
ファイルを削除しました。

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'messages#index'
+  resources :messages, only: [:index]
```
rootにあるのでこちらは不要です

shiratori-sh replied:
resources :messages, only: [:index]を削除しました。

shiratori-sh committed:
Edit haml files & messages_controller

mailer.html.haml,mailer.text.haml削除不要ルーティング削除
application.html.haml、index.html.hamlファイルの不要なハッシュロケット削除

TOMARUMARU reviewed (approved):
LGTM
